rpg-docs/Model/Character/Buffs.js

```diff
@@ -42,12 +42,18 @@ Schemas.Buff = new SimpleSchema({
 		allowedValues: _.pluck(colorOptions, "key"),
 		defaultValue: "q",
 	},
+	//the id of the feature, buff or item that created this buff
+	parent: {
+		type: Schemas.Parent,
+		optional: true,
+	},
 });
 
 Buffs.attachSchema(Schemas.Buff);
 
 Buffs.attachBehaviour("softRemovable");
-makeParent(Buffs, ["name", "enabled"]); //parents of effects
+makeParent(Buffs, ["name", "enabled"]); //parents of effects, attacks, proficiencies
+makeChild(Buffs, ["enabled"]); //children of lots of things //perhaps make "enabled" no longer a child now that we can individually toggle buffs?
```

Comment:
Do we want actually want Buffs.enabled to be a child? I'm not sure how useful that would be, or if it would just be an annoyance.